test/e2e/e2e_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"testing"
 
 	"k8s.io/kubernetes/test/e2e/framework"
+	_ "k8s.io/kubernetes/test/e2e/scheduling"
```

Comment:
@davidopp this is the key that makes it all work, it's binary smashing the scheduling tests into it.